Extensions/DataSources/src/DataSourcesHub/components/DataSourceDetails.tsx

```diff
@@ -1,14 +1,20 @@
 import * as React from "react";
 import { Header, TitleSize } from "azure-devops-ui/Header";
-import * as Reflux from 'reflux';
 import { TextField, TextFieldWidth } from "azure-devops-ui/TextField";
 import { DataSourceActionCreators } from "../action-creators/DataSourceActionCreators";
+import { DATA_SOURCE } from '../Resources/DataSourceResources';
```

Comment:
importing individual resources is not a good practice
instead import the Resource class and then use them like Resources.DataSource